daemon/cluster/executor/container/executor.go

```diff
@@ -5,6 +5,8 @@ import (
 	"sort"
 	"strings"
 
+	"sync"
```

Comment:
This `"sync"` line doesn't need to be separated from the other standard library imports with a blank line.

Reply:
fixed that, it was the IDE automatically adding that line..